scripts/getPastClientVersion.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('node:fs/promises');
```

Comment:
Just for understanding - what is this script needed for?

Reply:
It's for pulling down old client versions to test reverting locally. This originally needed to be unmerged because we didn't test the changes with an older client version, so it seemed appropriate to add the functionality here.

Reply:
sounds good - nice to be aware of - could be worth adding something to Confluence on how to do this/use this, a very useful little script!